jwst/nsclean/lib.py

```diff
@@ -91,7 +91,7 @@ def __init__(self, detector, mask, fc=1/(2*2048/16), kill_width=1/(2*2048/16)/4,
         FW = np.fft.rfft2(np.fft.ifftshift(W))
         with np.errstate(divide='ignore'):
             self.P = 1 / np.fft.irfft2(np.fft.rfft2(np.array(self.mask, dtype=np.float32)) * FW, (self.ny,self.nx))
-        self.P = np.where(self.mask==True, self.P, 0.) # Illuminated areas carry no weight
+        self.P = np.where(self.mask, self.P, 0.) # Illuminated areas carry no weight
```

Comment:
I'm pretty sure I tried making those same changes and found that it changed the results. So I restored the original and was just living with the codestyle errors for now.

Reply:
Or maybe I tried changing it to `[self.mask is True]`.  I'll do some more playing around with my local code and see if I can get it to work.

Reply:
Huh, OK testing in my local branch now shows that simply removing the `==True` seems to work just find and does not affect the results. So I'll add that to my #8111 PR that makes another little fix.